microservices.yml

```diff
@@ -496,6 +525,14 @@ jobs:
       - attach-built-docker-image
       - push-built-image-to-ecr
 
+  push-image-prod:
```

Comment:
The reason this is a separate job is to allow us to filter it within services to just tagged builds.